mos-platform/vic20/link-5k.ld

```diff
@@ -0,0 +1,23 @@
+/* VIC-20 PRG file linker script.
+ *
+ * BASIC memory regions are freely overwritten, but any KERNAL regions are left alone.
+ *
+ * Produces a PRG file with a SYS command to start the program.
+ */
+
+__basic_zp_start = 0x0000;
+__basic_zp_end = 0x0090;
+
+MEMORY {
+    ram (rw) : ORIGIN = 0x1001, LENGTH = 0x0DFF
```

Comment:
Could we have this symbol-controlled instead of having separate linker scripts? If there's only 3 cases the logic should be fairly straightforward.

Reply:
Should be fixed now.